lib/main.dart

```diff
@@ -29,13 +31,31 @@ class WasteExchange extends StatelessWidget {
       systemNavigationBarIconBrightness: Brightness.dark,
     ));
     return MaterialApp(
-        debugShowCheckedModeBanner: false,
-        title: Constants.APP_TITLE,
-        onGenerateRoute: Router.generateRoute,
-        theme: ThemeData(
-          appBarTheme: AppBarTheme(iconTheme: AppTheme.iconTheme),
-          textTheme: AppTheme.textTheme,
-        ),
-        home: MapScreen());
+      debugShowCheckedModeBanner: false,
+      title: Constants.APP_TITLE,
+      onGenerateRoute: Router.generateRoute,
+      theme: ThemeData(
+        appBarTheme: AppBarTheme(iconTheme: AppTheme.iconTheme),
+        textTheme: AppTheme.textTheme,
+      ),
+      supportedLocales: [ Locale('en', 'US'), Locale('ta', 'IN') ],
+      localizationsDelegates: [
+        AppLocalizations.delegate,
+        GlobalMaterialLocalizations.delegate,
```

Comment:
Should we move all localization related params into it's own class that gives us values for supportedLocales, localizationsDelegates, localeResolutionCallback

Reply:
This seems to be done